index.html

```diff
@@ -6341,19 +6341,18 @@ <h2>Revision History</h2>
   <section class="appendix">
     <h2>Acknowledgements</h2>
     <p>
-The Working Group extends deep appreciation and heartfelt thanks to our
-Chairs Brent Zundel and Dan Burnett, as well as our W3C Staff Contact, Ivan
-Herman, for their tireless work in keeping the Working Group headed in a
-productive direction and navigating the deep and dangerous waters of the
-standards process.
+The Working Group extends deep appreciation and heartfelt thanks to our Chairs
```

Comment:
Noticed it after you merged it...
```suggestion
The Working Group extends deep appreciation and heartfelt thanks to our Chairs,
```